lib/src/layer/polygon_layer/polygon_layer.dart

```diff
@@ -126,10 +152,41 @@ class _PolygonLayerState extends State<PolygonLayer> {
             )
             .toList();
 
+    final triangles = !widget.performantRendering
+        ? null
+        : List.generate(
+            culled.length,
+            (i) {
+              final culledPolygon = culled[i];
+              if (!culledPolygon.polygon.performantRendering) return null;
+
+              final points = culledPolygon.holePoints.isEmpty
+                  ? culledPolygon.points
+                  : culledPolygon.points
+                      .followedBy(culledPolygon.holePoints.expand((e) => e));
+
+              return Earcut.triangulateRaw(
+                List.generate(
+                  points.length * 2,
+                  (ii) => ii % 2 == 0
+                      ? points.elementAt(ii ~/ 2).x
+                      : points.elementAt(ii ~/ 2).y,
+                  growable: false,
+                ),
+                // Not sure how just this works but it seems to :D
+                holeIndices: culledPolygon.holePoints.isEmpty
+                    ? null
+                    : [culledPolygon.points.length],
```

Comment:
Are we fine with having `holeIndices` like it is now?

Reply:
If it ain't broke, don't fix it :D
Tbh, I still don't really understand why it works, and I think it might be a 'feature' of dart_earcut that doesn't exist in the originals.